spk/syncthing-inotify/Makefile

```diff
@@ -0,0 +1,37 @@
+SPK_NAME = syncthing-inotify
+SPK_VERS = 0.8.2
+SPK_REV = 1
+SPK_ICON = src/${SPK_NAME}.png
+BETA = 1
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
+
+DEPENDS = cross/busybox cross/$(SPK_NAME)
+SPK_DEPENDS = syncthing
+START_DEP_SERVICES = syncthing
```

Comment:
In general, yes. Otherwise syncthing-inotify will generate a lot of stuff like:

> [WARNING] 23:38:46 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused
[WARNING] 23:38:51 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused
[WARNING] 23:38:56 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused
[WARNING] 23:39:01 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused
[WARNING] 23:39:06 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused
[WARNING] 23:39:11 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused
[WARNING] 23:39:16 Failed to perform request Get http://192.168.0.33:7070/rest/events?since=0: dial tcp 192.168.0.33:7070: getsockopt: connection refused

it will try to connect to syncthing every 5 seconds. Does a user need that?